spec/requests/api/arbitration_default_spec.rb

```diff
@@ -0,0 +1,140 @@
+RSpec.describe 'Arbitration Default API' do
+  let(:ems) { FactoryGirl.create(:ext_management_system) }
+
+  context 'arbitration defaults index' do
+    it 'can list the arbitration defaults' do
+      FactoryGirl.create(:arbitration_default, :ems_id => ems.id)
+
+      api_basic_authorize collection_action_identifier(:arbitration_defaults, :read, :get)
+      run_get(arbitration_defaults_url)
+      expect_query_result(:arbitration_defaults, 1, 1)
+    end
+  end
+
+  context 'arbitration defaults create' do
+    let(:request_body) do
+      {:ems_id => ems.id}
+    end
+
+    it 'rejects creation without appropriate role' do
+      api_basic_authorize
+
+      run_post(arbitration_defaults_url, request_body)
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it 'supports single arbitration_default creation' do
+      api_basic_authorize collection_action_identifier(:arbitration_defaults, :create)
+
+      expect do
+        run_post(arbitration_defaults_url, request_body)
+      end.to change(ArbitrationDefault, :count).by(1)
+      expect(response).to have_http_status(:ok)
+
+      id = response_hash['results'].first['id']
+      expect(ArbitrationDefault.exists?(id)).to be_truthy
```

Comment:
Now that you've verified that you created an `ArbitrationDefault` in the request, I'd feel reasonbly confident that it was the one you got back in the response, but it could be a matter of taste whether you leave this additional check in here